cassandra-dac/src/main/java/org/sunbird/cassandra/CassandraOperation.java

```diff
@@ -60,6 +60,15 @@
   public void deleteRecord(
       String keyspaceName, String tableName, Map<String, String> compositeKeyMap);
 
+  /**
+   * @desc This method is used to delete record in cassandra db by their primary key(identifier)
```

Comment:
@desc Delete one or more records from Cassandra DB corresponding to given list of primary keys..